_layouts/post.html

```diff
@@ -44,5 +44,29 @@ <h2 id="references">References</h2>
 	{% bibliography --cited %}
   </div>
   {% endif %}
+  <details>
+	<summary>Citing this Post</summary>
+	<div class="highlighter-rouge">
+		<div class="highlight">
+			<pre class="highlight"><code id="citation-code">@misc{% raw %}{{% endraw %}{{ page.slug  }},
+	author = "{% for c in page.contributors %}{{ site.data.CONTRIBUTORS[c].name }}{% if forloop.last == false %} and {% endif %}{% endfor %}",
+	title = "{{ page.title }}",
+	year = "{{ page.date | date: "%Y"}}",
+	month = "{{ page.date| date: "%m"}}",
+	day = "{{ page.date| date: "%d" }}"
+	url = "{% raw %}\url{{% endraw %}{{ site.url }}{{ site.baseurl }}{{page.url}}{% raw %}}{% endraw %}",
```

Comment:
```suggestion
	url = "{% raw %}{% endraw %}{{ site.url }}{{ site.baseurl }}{{page.url}}{% raw %}{% endraw %}",
```

Reply:
I am not sure if I removed the correct closing bracket here.. too many of them..

Reply:
you got the correct ones, the diff looks odd but that's down to diffing algorithms. but we can simplify it now that we don't need a literal } anymore:

```suggestion
	url = "{{ site.url }}{{ site.baseurl }}{{page.url}}",
```